pcsx2/USB/usb-pad/usb-pad.h

```diff
@@ -435,16 +455,16 @@ namespace usb_pad
 	static const int HATS_8TO4[] = {PAD_HAT_N, PAD_HAT_E, PAD_HAT_S, PAD_HAT_W};
 
 #define PAD_VID 0x046D
-#define PAD_PID 0xCA03     //black MOMO
+#define PAD_PID 0xCA03 //black MOMO
 #define GENERIC_PID 0xC294 //actually Driving Force aka PID that most logitech wheels initially report
 #define PID_DF 0xC294
 #define PID_DFP 0xC298 //SELECT + R3 + RIGHT SHIFT PADDLE (R1) ???
 #define PID_DFGT 0xC29A
 #define PID_FORMULA 0xC202 //Yellow Wingman Formula
-#define PID_FGP 0xC20E     //Formula GP (maybe GT FORCE LPRC-1000)
-#define PID_FFGP 0xC293    // Formula Force GP
-#define PID_GTF 0xC293     // as is Formula Force GP
-#define PID_G25 0xC299     // OutRun 2 (jp) supports it apparently
+#define PID_FGP 0xC20E //Formula GP (maybe GT FORCE LPRC-1000)
```

Comment:
Keep these clang-format changes?

Reply:
You can keep them, since not all the comments are aligned, or you can align them all.